shell/platform/android/io/flutter/plugin/editing/ListenableEditingState.java

```diff
@@ -52,6 +52,41 @@ void didChangeEditingState(
 
   private BaseInputConnection mDummyConnection;
 
+  // Fields for delta.
+  private String oldText;
+  private String deltaText;
+  private String deltaType;
+  private int deltaStart;
+  private int deltaEnd;
```

Comment:
Would it be cleaner to have a separate TextEditingDelta class?

Reply:
Definitely!